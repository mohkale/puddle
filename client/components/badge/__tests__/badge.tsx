import React from 'react';
import renderer from 'react-test-renderer';

import Badge from '../index';

describe('badge', () => {
  it('can render', () => {
    const tree = renderer
      .create(<Badge num={5} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('can render even without a numerical value', () => {
    const tree = renderer
      .create(<Badge />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
