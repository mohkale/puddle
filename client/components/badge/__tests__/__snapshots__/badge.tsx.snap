// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`badge can render 1`] = `
<span
  className="badge"
>
  5
</span>
`;

exports[`badge can render even without a numerical value 1`] = `
<span
  className="badge"
>
  ?
</span>
`;
